expression/integration_test.go

```diff
@@ -7667,6 +7675,14 @@ func (s *testIntegrationSuite) TestIssue17287(c *C) {
 	tk.MustQuery("execute stmt7 using @val2;").Check(testkit.Rows("1589873946"))
 }
 
+func (s *testIntegrationSuite) TestIssue26989(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
```

Comment:
Need to enable new collation

Reply:
nope, you can see the bug https://github.com/pingcap/tidb/issues/26989#issuecomment-895713756 the `new collation` feature is disabled but still case insensitive.